api/src/Entity/ContentNode.php

```diff
@@ -46,6 +47,7 @@
 #[ORM\Entity(repositoryClass: ContentNodeRepository::class)]
 #[ORM\InheritanceType('SINGLE_TABLE')]
 #[ORM\DiscriminatorColumn(name: 'strategy', type: 'string')]
+#[ORM\UniqueConstraint(name: 'contentnode_parentid_slot_position_unique', columns: ['parentid', 'slot', 'position'])]
```

Comment:
For as long as we use Gedmo/Sortable this seems to be fine, as we have `#[Gedmo\SortableGroup]` on both parent and slot. So the uniqueness should be ensured/fixed automatically by the Sortable extension.

If at any point we remove Gedmo/Sortable and leave the sorting to the frontend, I would vote for not having this unique constraint. Further below we also documented as following:

> The API does not guarantee the uniqueness of parent+slot+position

Reply:
ack